<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DMeasure</class>
 <widget class="QDialog" name="DMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measurement input</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>207</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Detector name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabDet">
        <property name="text">
         <string>Detector</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sample name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabName">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Measurement time (in min):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabTime">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Sample volume (in ml):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabVol">
        <property name="text">
         <string>Volume</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Do you want to start measurement?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="MeasureOK">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>MeasureOK</sender>
   <signal>rejected()</signal>
   <receiver>DMeasure</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MeasureOK</sender>
   <signal>accepted()</signal>
   <receiver>DMeasure</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
